homeassistant/helpers/label_registry.py

```diff
@@ -96,7 +97,8 @@ class LabelRegistry:
     def __init__(self, hass: HomeAssistant) -> None:
         """Initialize the label registry."""
         self.hass = hass
-        self._store = hass.helpers.storage.Store(
+        self._store: Store[dict[str, list[dict[str, Any]]]] = Store(
```

Comment:
Can we use this type annotation?

https://github.com/home-assistant/core/blob/8809fa9b10dea7fb2f157bcb386413e472cc6b37/homeassistant/helpers/label_registry.py#L254